package/CODE_OF_CONDUCT.md

```diff
@@ -0,0 +1,118 @@
+Therefore, we have a few ground rules we ask that our community members adhere
+to.
+
+Fundamentally, we are committed to providing a productive,
+harassment-free environment for everyone. Rather than considering this
+code an exhaustive list of things that you can’t do, take it in the
+spirit it is intended - a guide to make it easier to enrich all of us
+and the communities in which we participate.
+
+This code applies equally to founders, developers, mentors and new
+community members, in all spaces managed by MDAnalysis. This
+includes the mailing lists, our GitHub organizations, our chat rooms,
+in-person events, and any other forums created by the project team. In
+addition, violations of this code outside these spaces may affect a
+person's ability to participate within them.
+
+If you believe someone is violating the code of conduct, we ask that you report
+it by
+emailing [*User-mailling-list*](mailto:mdnalysis-discussions@googlegroups.com)
+or in anonymous by filling out [*this form*](TODO).
+
+By embracing the following principles, guidelines and actions to follow or
+avoid, you will help us make MDAnalysis a welcoming and productive community. If
+that doesn't answer your questions, feel free to contact us
+at [*User-mailing-list*](mailto:mdnalysis-discussions@googlegroups.com).
+
+
+1. **Be friendly and patient**.
+
+2. **Be welcoming**. We strive to be a community that welcomes and supports
+   people of all backgrounds and identities. This includes, but is not limited
+   to, members of any race, ethnicity, culture, national origin, color,
+   immigration status, social and economic class, educational level, sex, sexual
+   orientation, gender identity and expression, age, physical appearance, family
+   status, political belief, technological or professional choices, academic
+   discipline, religion, mental ability, and physical ability.
+
+3. **Be considerate**. Your work will be used by other people, and you in turn
+   will depend on the work of others. Any decision you take will affect users
+   and colleagues, and you should take those consequences into account when
+   making decisions. Remember that we're a world-wide community. You may be
+   communicating with someone with a different primary language or cultural
+   background.
+
+4. **Be respectful**. Not all of us will agree all the time, but disagreement is
+   no excuse for poor behavior or poor manners. We might all experience some
+   frustration now and then, but we cannot allow that frustration to turn into a
+   personal attack. It’s important to remember that a community where people
+   feel uncomfortable or threatened is not a productive one.
+
+5. **Be careful in the words that you choose**. Be kind to others. Do not insult
+   or put down other community members. Harassment and other exclusionary
+   behavior are not acceptable. This includes, but is not limited to:
+   * Violent threats or violent language directed against another person
```

Comment:
A threat does not have to be violent to be unacceptable. I would just say "Threats or violent language...".